src/main/java/com/bootcamp/demo/validation/UserValidator.java

```diff
@@ -0,0 +1,109 @@
+package com.bootcamp.demo.validation;
+
+import com.bootcamp.demo.model.User;
+
+public class UserValidator {
+
+    public final static String NAME_PATTERN = "[A-Z]+[a-z]+[A-Z]*[a-z]*";
+    public final static String EMAIL_PATTERN = "[a-zA-Z0-9._-]+@[a-zA-Z]+[.][a-zA-Z]+";
+    public final static String CONTAINING_UPPERCASE_PATTERN = ".*[A-Z]+.*";
+    public final static String CONTAINING_LOWERCASE_PATTERN = ".*[a-z]+.*";
+    public final static String CONTAINING_DIGIT_PATTERN = ".*[0-9]+.*";
+    public final static String CONTAINING_SPECIAL_CHARACTER_PATTERN = ".*[!#$%^&*~_=+./<>-]+.*";
+    public final static String PHONE_NUMBER_PATTERN = "[+]40[0-9]{9}";
```

Comment:
The correct order is `public static final String` - please reorder the modifiers.